### DISCUSSION TITLE
How to use Cambria Math font

### DISCUSSION THREAD

**Discussion by Petrik1001** [ManimCommunity/manim#2599] [General]
## Description of proposed feature
How to use Cambria Math font?

**Comment by behackl** [comment]
Hello! I have converted the issue to a discussion, which seems more appropriate at this point. In what capacity exactly would you like to use Cambria Math? Cambria Math could be seen as a woraround for "normal" text rendering (i.e., with `Text`) in settings where LaTeX is not available. Manim can directly interact with LaTeX though using `Tex` and `MathTex`, so I am not quite sure in which capacity you would like to use Cambria. Please give more details.
